client/wfnews-war/src/main/angular/src/app/components/draggable-panel/draggable-panel.component.ts

```diff
@@ -186,14 +197,11 @@ export class DraggablePanelComponent implements OnInit, OnChanges, OnDestroy {
       } else {
         //identify anything other than incident
         if (
-          this.identifyItem.layerId.includes('bans-and-prohibitions') ||
-          this.identifyItem.layerId.includes('evacuation-orders-and-alerts') ||
-          this.identifyItem.layerId.includes('area-restrictions') ||
-          this.identifyItem.layerId.includes('weather-stations')
+          ['bans-and-prohibitions', 'evacuation-orders-and-alerts', 'area-restrictions', 'weather-stations', 'danger-rating'].some(str => this.identifyItem.layerId.includes(str))
```

Comment:
You don't need the `some` here. Just `array.includes(layerId)` will return a true/false if the string is in the array (the same as the list with or operators)

`['bans-and-prohibitions', 'evacuation-orders-and-alerts', 'area-restrictions', 'weather-stations', 'danger-rating'].includes(this.identifyItem.layerId)`

Reply:
The reason I use "some" here is because we have layers end with "bans-and-prohibitions-cat1" or "bans-and-prohibitions-cat2" etc.  So I am checking if the layerID contains the strings I list there